### PR TITLE
Start 2024 advertisement

### PR DESCRIPTION
**Pull request by jomey**
This one has no preview, so it is best to check out locally

### CODE REVIEW

File: cookiecutter.yaml [hunk 1]
```diff
@@ -1,23 +1,34 @@
...
+    <ul>
+      <li>Project Coordinator</li>
+      <li>Project Lead</li>
+      <li>Project Helper</li>
+    </ul>
+    Visit the <a target=\"_blank\" href=\"https://escience.washington.edu/using-data-science/hackweeks/hackweek-organizer-training/\">
+    Hackweek Organizer Training page<a/> to learn more about these roles or reach
+    out to us
+    "
+  contact_email: snowex_hackweek@cloud.washington.edu
```
Comment (hackweek-admin) [inline cookiecutter.yaml:22]: is this email correct?

**Review by hackweek-admin — APPROVED** [review]
Added the preview! Looks good. Could now acknowledge NASA funds under sponsors?

**Comment by jomey** [comment]
> Added the preview! Looks good. Could now acknowledge NASA funds under sponsors?

Thank you, Scott!

File: cookiecutter.yaml (hunk 1 above)
Comment (jomey) [reply cookiecutter.yaml:22]: That is a question I forgot to ask yesterday, let me follow up on Slack

**Comment by aaarendt** [comment]
@jomey we can have an organizing team application form ready very soon for you to link to. I think Mark has been working on this.

**Comment by jomey** [comment]
> @jomey we can have an organizing team application form ready very soon for you to link to. I think Mark has been working on this.

Sounds good. I will hold off on this PR until the form can be published.

I have a tentatively date written down for a SnowEx community call on 02/15. Could be a good target date for having all of this online.

File: cookiecutter.yaml (hunk 1 above)
Comment (JessicaS11) [reply cookiecutter.yaml:24]:
```suggestion
```

Comment (jomey) [reply cookiecutter.yaml:24]: This line can be removed and is no longer in use

**Review by JessicaS11 — APPROVED** [review]

**Review by aaarendt — APPROVED** [review]

**Pull request merged**